packages/react/menu/src/Menu.stories.tsx

```diff
@@ -466,7 +455,7 @@ const itemClass = css({
 });
 
 const subTriggerClass = css(itemClass, {
-  '&[data-state="open"]': {
+  '&:not(:focus)[data-state="open"]': {
```

Comment:
This is the only meaningful change I had to make after checking every single story manually. It seems we were relying on some specificity bug from stitches before.